script.trakt/resources/settings.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
wondering if we should not start to request developers to adopt the new settings format which is supported since leia. What do you think? It is not documented on the wiki yet (I plan to create a new page for it when I have some time) but it is quite straightforward (https://github.com/xbmc/xbmc/blob/master/system/settings/settings.xml)

Reply:
Also, shouldn't we bump the python api for leia as well? There are new methods that will only work in leia and others that have been removed. However `2.25.0` will be installed (manually) in both Krypton and Leia.
@Razzeee

Reply:
Documentation is needed and it would be even better to have some kind of automatic conversion.

Bumping sounds like a good idea, but I'm not the export on that.